automation/test.sh

```diff
@@ -54,6 +54,10 @@ fi
 
 export KUBEVIRT_NUM_NODES=2 
 
+if [[ "$TARGET" =~ okd-.* ]]; then
+  export KUBEVIRT_NUM_NODES=3
```

Comment:
Is there a difference from okd 4.1 to 4.2, as in one counts master as node and the other one not?

Reply:
No, the problem is that for OKD 4.2 there are two workers required, otherwise the cluster will not come up. I was told that this would also be the case for 4.1, so I made this general.

Reply:
Erm, as I think about what you wrote: yes. :-$